readme.md

```diff
@@ -0,0 +1,175 @@
+﻿# CargoDash Usage Guide
+This document will help you to understand to make use of CargoDash. CargoDash is a diagnostics tool for developing autonomous vehicles and is written in Python code (version 3.8.5). For any questions please feel free to create an issue.
+
+## Python Dependencies
+```
+    aiohttp v3.7.3
+    can v3.3.4
+    canopen v1.1.0
+    numpy v1.19.4
+    pyyaml v5.3.1
+```
+## Installation
+1. Clone the repository
+```
+    git clone https://github.com/ZeroPoint095/CargoDash
+```
+2. Install the Python dependencies with pip3
+```
+    pip3 install -r requirements.txt
+```
+3. Run startup.sh to use CargoDash
+```
+    ./startup.sh
+```  
+
+## Introduction
+CargoDash is a tool that listens to incoming messages and sets the values of these messages into user-defined nodes. These user-defined nodes can be requested with CargoDash's HTTP-server. Besides this, it logs all raw incoming messages into a buffered logger. 
+
+For now, CargoDash works only with CanOpen messages but with the architecture of CargoDash, you should be able to add your desired message protocol. 
+
+The logger can be requested to log all messages from the buffer on command. For example, you can request the buffer on a dangerous event or when the user wants it.
```

Comment:
Ik zou hier iets meer uitleg over geven. Dat het een circular buffer is die continu aan het opnemen is. Dan, bij bijvoorbeeld een error, wordt de buffer naar een log geschreven